AirCasting/Map/AirMapView.swift

```diff
@@ -15,6 +15,7 @@ struct AirMapView: View {
     @StateObject var statsContainerViewModel: StatisticsContainerViewModel
     @StateObject var mapStatsDataSource: MapStatsDataSource
     @ObservedObject var session: SessionEntity
+    @Binding var showLoadingIndicator: Bool
```

Comment:
Let's not change it, but for the record - I think that showing a loading indicator should not be implemented in the `AirMapView` itself. Switching between loading indicator and map view should be performed higher in the view hierarchy because loading state not an intrinsic part of the map view itself, it's a higher level concept.